src/libraries/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.OSX.cs

```diff
@@ -199,12 +199,12 @@ public static void ScheduleEventStream(SafeEventStreamHandle eventStream)
                         Debug.Assert(s_scheduledStreamsCount == 0);
                         s_scheduledStreamsCount = 1;
                         var runLoopStarted = new ManualResetEventSlim();
-                        new Thread(args =>
+                        new Thread(static args =>
                         {
                             object[] inputArgs = (object[])args!;
                             WatchForFileSystemEventsThreadStart((ManualResetEventSlim)inputArgs[0], (SafeEventStreamHandle)inputArgs[1]);
                         })
-                        { IsBackground = true }.Start(new object[] { runLoopStarted, eventStream });
+                        { IsBackground = true }.UnsafeStart(new object[] { runLoopStarted, eventStream });
```

Comment:
Can we name it too while you're here? 
BTW, why does Linux FSW use Task.Factory.Start(..LongRunning..) but OSX starts a thread?

Reply:
> Can we name it too while you're here?

There are other threads started by the libraries besides these.  I'd rather someone do an audit specifically for naming and ensure names are consistent, rather than doing a one-off here.  I only touched this one because it could safely use UnsafeStart because ExecutionContext is flowed differently.

> BTW, why does Linux FSW use Task.Factory.Start(..LongRunning..) but OSX starts a thread?

They were implemented by two different people?

Reply:
Fair enough.